e2e-tests/IIoTPlatform-E2E-Tests/TestHelper.cs

```diff
@@ -274,5 +276,59 @@ internal static class TestHelper {
             bool isSuccess = json.isSuccess;
             Assert.True(isSuccess);
         }
+
+        /// Wait for all API microservices of IIoT platform to be healthy.
+        /// </summary>
+        /// <param name="config"> IIoT platform configuration </param>
+        /// <param name="ct"> Cancellation token </param>
+        /// <returns></returns>
+        public static async Task WaitForServicesAsync(
+            IIIoTPlatformConfig config,
+            CancellationToken ct
+        ) {
+            const string healthyState = "Healthy";
+
+            var healthRoutes = new string[] {
+                TestConstants.APIRoutes.RegistryHealth,
+                TestConstants.APIRoutes.PublisherHealth,
+                TestConstants.APIRoutes.TwinHealth,
+                TestConstants.APIRoutes.HistoryHealth,
+                TestConstants.APIRoutes.GatewayHealth,
+                TestConstants.APIRoutes.VaultHealth,
+                TestConstants.APIRoutes.EventsHealth,
+                TestConstants.APIRoutes.JobOrchestratorHealth
+            };
+
+            var client = new RestClient(config.BaseUrl) {
+                Timeout = 60 * 1000
+            };
+
+            while(true) {
+                ct.ThrowIfCancellationRequested();
+
+                var tasks = new List<Task<IRestResponse>>();
+
+                foreach(var healthRoute in healthRoutes) {
+                    var request = new RestRequest(Method.GET) {
+                        Resource = healthRoute
+                    };
+
+                    tasks.Add(client.ExecuteAsync(request, ct));
+                }
+
+                Task.WaitAll(tasks.ToArray());
+
+                var healthyServices = tasks
+                    .Where(task => task.Result.StatusCode == System.Net.HttpStatusCode.OK)
+                    .Where(task => task.Result.Content == healthyState)
+                    .Count();
+
+                if (healthyServices == healthRoutes.Length) {
+                    return;
```

Comment:
Please add some success logging, that all services marked as `Healthy`

Reply:
You mean a comment ?

Reply:
Done.